packages/website/README.md

```diff
@@ -2,6 +2,7 @@
 
 Built using [Docusaurus 2](https://v2.docusaurus.io/).
 
+
```

Comment:
Added here to trigger website deployment after merge

Reply:
Can we add `.github/workflows/website-deployment.yml` to paths instead to make sure it's always run if the pipeline changes?